### DISCUSSION TITLE
Welcome to CubeSatSim Discussions!

### DISCUSSION THREAD

**Discussion by alanbjohnston** [alanbjohnston/CubeSatSim#135] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MikeMcPherson** [comment]
Mike McPherson, KQ9P, trustee for W4UVA at the University of Virginia.  Retired higher ed IT guy.  I volunteer with the Aerospace Engineering Spacecraft Design program, and built the ground station for our first orbital mission, the Virginia Cubesat Constellation.  I am building two CubeSat Simulators for use in the capstone course to give the students hands-on experience in between missions.

**Comment by W7DGP** [comment]
After getting the AMSAT ground station software working on a Pi 4 I turned to organizing the demo process. So here is my setup. I purchased a low-cost travel router (Opal GL-SFT1200 on Amazon) and set it up as a standard Wifi router access point. Once configured, it sends out its SSID and default password. Next, I brought up the groundstation Raspberry Pi 4 and changed the Wifi to the Opal router. It locked in nicely. Then I powered everything down and removed the keyboard, mouse and monitor from the ground station Pi 4. Going headless, right? The next step is going to be the normal demo mode. Everything is powered up except the Cubesat Sim. I changed my laptop to the Opal router network and used VNC to log into the ground station and I see Fox Telem looking for a signal! I started the Cubesat Sim and soon I'm seeing FSK data and a nice eye diagram.  Using my rather large USB battery supply for the Opal router, I can run the demo totally on batteries anywhere inside or outside. I can log in multiple laptops to the ground station Pi over this Wifi. This went together quite easily. 
-Larry W7DGP

**Comment by davidvine1999** [comment]
Greetings from a fellow Radio Amateur (David Vine, WA1EAW) who follows the AMSAT Education program and and other AMSAT projects. I do so to write articles about them for AMSAT publication. My passion as a member of the Amateur Radio Service (FCC term) is learning, pure and simple. My focus is on the [CubeSat Simulator](https://cubesatsim.org/)  (CSS) and it's precursors. These include the [Fox in a Box](https://cubesatsim.org/download/How%20To%20Set%20Up%20Fox-in-a-Box-V4.pdf) (PDF) Linux distribution; CSS Lite [software;](https://satsearch.co/products/amsat-cubesat-simulator-lite) and the CSS Lite [board](https://cubesatsim.org/download/cubesatsim-lite-readme.pdf) (PDF) with low power transceiver. Here is a [[video]](https://www.youtube.com/watch?v=bk_Y9IKmYio) about FiaB by Professor Alan Johnston.
